tutorials/deploy-mean-app-mongodb-replication.md

```diff
@@ -0,0 +1,205 @@
+
+        sudo su - bitnami
+
+1. Navigate to the default application folder:
+
+        cd /opt/bitnami/apps/
+
+1. Create the folder structure for the MEAN application:
+
+        sudo mkdir myapp
+        sudo mkdir myapp/conf
+        sudo mkdir myapp/htdocs
+
+1. Download the application source code into the `myapp/htdocs` folder:
+
+        cd myapp/htdocs
+        sudo git clone https://github.com/cornflourblue/mean-stack-registration-login-example.git .
+
+1. Install the necessary application dependencies:
+
+        sudo npm install
+
+1. Login to the MongoDB database using the `mongo` command-line client. Use the password obtained from the [Deployment Manager](https://console.cloud.google.com/dm/deployments) when prompted.
+
+        mongo admin --username root -p
+
+1. Once logged in, create a new database and user account for the application:
+
+        use mydb
+        db.createUser(
+           {
+             user: "myuser",
+             pwd: "mypass",
+             roles: [ "readWrite" ]
+           }
+        )
+        exit
+
+1. Modify the application configuration to use the new database and credentials by modifying the `/opt/bitnami/apps/myapp/htdocs/config.json` file as shown below:
+
+        {
+            "connectionString": "mongodb://myuser:mypass@localhost:27017/mydb",
+            ...
+        }
+
+## Serve the application with Apache
+
+Express will typically serve the MEAN application on port 3000. This port is closed by default in Bitnami MEAN for security reasons. However, Bitnami MEAN also includes the Apache Web server, which can be configured as a reverse proxy for Express on the standard Web server port 80.
+
+Configure Apache to work as a reverse proxy for Express and accept requests for the MEAN application on port 80:
+
+1. Create the `/opt/bitnami/apps/myapp/conf/httpd-prefix.conf` file and add the line below to it:
+
+        Include "/opt/bitnami/apps/myapp/conf/httpd-app.conf"
+
+1. Create the `/opt/bitnami/apps/myapp/conf/httpd-app.conf` file and add the content below to it to enable the reverse proxy:
+
+        ProxyPass / http://127.0.0.1:3000/
+        ProxyPassReverse / http://127.0.0.1:3000/
+
+1. Add the following line to the end of the main Apache configuration file at `/opt/bitnami/apache2/conf/bitnami/bitnami-apps-prefix.conf`:
+
+        Include "/opt/bitnami/apps/myapp/conf/httpd-prefix.conf"
+
+1. Restart the Apache server using the Bitnami control script:
+
+        sudo /opt/bitnami/ctlscript.sh restart apache
+
+1. Start the Express server:
+
+        cd /opt/bitnami/apps/myapp/htdocs
+        forever start server.js
+
+Browse to the public IP address of your Bitnami MEAN instance and confirm that you see the MEAN example application's welcome page. Register a new user account in the example application using the `Register` link and then log in to the example application by entering the new user account credentials into the login form and clicking the `Login` button.
```

Comment:
Style this and following references to UI text as bold  rather than code (`Register` -> **Register**)